packages/reports/tests/integration/components/power-select-bulk-import-trigger-test.js

```diff
@@ -45,7 +45,7 @@ module('Integration | Component | power select bulk import trigger', function(ho
     assert.dom('.ember-power-select-multiple-options').exists('The component renders');
   });
 
-  test('paste to trigger bulk import', async function(assert) {
+  skip('paste to trigger bulk import', async function(assert) {
```

Comment:
does this not work now?

Reply:
yup same as in master branch.

Reply:
Yeah. I misunderstood why master branch isn't working. I got this to work.